samples/camera/index.html

```diff
@@ -5,68 +5,98 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
   <title>Image Capture Capabilities</title>
-  <link rel="stylesheet" href="../css/style.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
I understand this is the recommended way, but we need to account for offline cases also .. Just add a note saying that we will look into self-hosting.